vast/integration/reference/node-zeek-dns-log/step_04.ref

```diff
@@ -0,0 +1 @@
+0
```

Comment:
Can we adjust the baseline to be non-zero? Otherwise it's difficult to separate faulty behavior from expected.

Reply:
The idea is to prove that this does not return the same result as the `:port` query in the previous step.

Reply:
Ah, that makes sense. A comment about that would be nice.